frontend/system/src/Admin/DrugEditForm/categories.ts

```diff
@@ -0,0 +1,85 @@
+// TODO: store this as editable config
+export const categories = {
+  routes: [
+    { label: 'Buccal', value: 'Buccal' },
+    { label: 'Dental', value: 'Dental' },
+    { label: 'Ear', value: 'Ear' },
+    { label: 'Eye', value: 'Eye' },
+    { label: 'Inhalation', value: 'Inhalation' },
+    { label: 'Intratracheal', value: 'Intratracheal' },
+    { label: 'Intrauterine', value: 'Intrauterine' },
+    { label: 'Nasal', value: 'Nasal' },
+    { label: 'Oral', value: 'Oral' },
+    { label: 'Parenteral', value: 'Parenteral' },
+    { label: 'Rectal', value: 'Rectal' },
+    { label: 'Subdermal', value: 'Subdermal' },
+    { label: 'Sublingual', value: 'Sublingual' },
+    { label: 'Topical', value: 'Topical' },
+    { label: 'Transdermal', value: 'Transdermal' },
+    { label: 'Vaginal', value: 'Vaginal' },
+  ],
+  forms: [
+    { label: 'Aerosol pressurised', value: 'Aerosol pressurised' },
+    { label: 'Capsule', value: 'Capsule' },
+    { label: 'Cream', value: 'Cream' },
+    { label: 'Device', value: 'Device' },
+    { label: 'Drop', value: 'Drop' },
+    { label: 'Emulsion', value: 'Emulsion' },
+    { label: 'Film', value: 'Film' },
+    { label: 'Foam', value: 'Foam' },
+    { label: 'Gas', value: 'Gas' },
+    { label: 'Gel', value: 'Gel' },
+    { label: 'Granules', value: 'Granules' },
+    { label: 'Implant', value: 'Implant' },
+    { label: 'Liquid', value: 'Liquid' },
+    { label: 'Lotion', value: 'Lotion' },
+    { label: 'Lozenge', value: 'Lozenge' },
+    { label: 'Nebule', value: 'Nebule' },
+    { label: 'Ointment', value: 'Ointment' },
+    { label: 'Paint', value: 'Paint' },
+    { label: 'Patch', value: 'Patch' },
+    { label: 'Paste', value: 'Paste' },
+    { label: 'Pessary', value: 'Pessary' },
+    { label: 'Powder for reconstitution', value: 'Powder for reconstitution' },
+    { label: 'Ring', value: 'Ring' },
+    { label: 'Shampoo', value: 'Shampoo' },
+    { label: 'Solution', value: 'Solution' },
+    { label: 'Solution (Intramuscular)', value: 'Solution (Intramuscular)' },
+    { label: 'Solution (Subcutaneous)', value: 'Solution (Subcutaneous)' },
+    { label: 'Spray', value: 'Spray' },
+    { label: 'Suppository', value: 'Suppository' },
+    { label: 'Suspension', value: 'Suspension' },
+    { label: 'Tablet', value: 'Tablet' },
+    { label: 'Tablet (Chewable)', value: 'Tablet (Chewable)' },
+    { label: 'Tablet (Crushable)', value: 'Tablet (Crushable)' },
+    { label: 'Tablet (Dispersible)', value: 'Tablet (Dispersible)' },
+    { label: 'Tablet (Enteric-coated)', value: 'Tablet (Enteric-coated)' },
```

Comment:
Have added some dose form qualifications here, as asked by Louisa to have them merged in with the dose form options, but will need to have her/someone go through to determine which are valid dose form qualifications for particular dose forms!

Reply:
I think the custom comments/details sounds like the way to go, as long as that resonates with Louisa too... referring to that SOP Rowena wrote, it looks like it was at least intended that there would only be a predefined set of dose forms and qualifications, with additions to be reviewed first. I'm assuming those additional form qualifications for the vaccines came about because there was nowhere better to store that information? Though we'll need to think about what we do with the existing data that looks like that if we go the comments route...

And yep managing the options in the backend for sure!

Reply:
Created #452 for this!